tfrog-motordriver/main.c

```diff
@@ -586,7 +586,7 @@ int main()
 
   // Enable watchdog
   printf("Watchdog init\n\r");
-  AT91C_BASE_WDTC->WDTC_WDMR = AT91C_WDTC_WDRSTEN | 0xFF00FF;  // 1s
+  AT91C_BASE_WDTC->WDTC_WDMR = AT91C_WDTC_WDRSTEN | 0x7d007d0;
```

Comment:
In my understanding, watchdog count clock is 32.768kHz divided by 128 = 256 Hz. (datasheet `15. Watchdog Timer (WDT)`)
0x7d0 means 7.8 seconds?

Reply:
that's my bad. will revert it back.